app/assets/javascripts/components/story/task/Task.js

```diff
@@ -2,30 +2,35 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { taskPropTypesShape } from '../../../models/beta/task';
 
-const Task = ({ task, onDelete, onToggle }) => (
+const Task = ({ task, onDelete, onToggle, disabled }) => (
   <div className="task">
     <label>
       <input
         type='checkbox'
         defaultChecked={task.done}
         onClick={onToggle}
+        disabled={disabled}
       />
       {task.name}
     </label>
-    <span
-      title={I18n.t('delete')}
-      className='delete-btn'
-      onClick={onDelete}
-    >
-      { I18n.t('delete') }
-    </span>
+    {
+      !disabled &&
```

Comment:
Can we add parentheses around the span here? Just to make it easier to read.

Reply:
Yeah, there are some other files with similar code, looking into it rn